_source/_assets/css/base/_elements.scss

```diff
@@ -155,18 +155,59 @@ pre {
   line-height: 1.25;
   padding:  0 0.1em;
   background-color: $color-tint;
+  direction: ltr;
+	text-align: left;
+	white-space: pre;
+	word-spacing: normal;
+	word-break: normal;
+	-moz-tab-size: 4;
+	-o-tab-size: 4;
+	tab-size: 4;
+	-webkit-hyphens: none;
+	-moz-hyphens: none;
+	-ms-hyphens: none;
```

Comment:
Tabs vs. spaces? There are a few more like this too.

Reply:
Copy paste from the syntax highlighter style sheet. Just pushed a fix.